lib/bone.js

```diff
@@ -1070,7 +1070,12 @@ class Bone {
     // find(1)
     // find([ 1, 2, 3 ])
     if (Array.isArray(conditions) || conditionsType == 'number') {
-      spell = new Spell(this);
+      if (values.length == 1 && typeof values[0] === 'object') {
+        // pass options to spell
+        spell = new Spell(this, values[0]);
+      } else {
+        spell = new Spell(this);
+      }
```

Comment:
similar checks already occur in the else branch at L1081, I think the outer if needs some refactor